components/page/home/Intro.tsx

```
@@ -8,195 +8,195 @@ import * as THREE from 'three';
+    // 회전 속도 상태를 설정
+    const rotationSpeed = hoveredFace ? 0.001 : 0.002; // 호버 시 속도를 느리게
+
+    // useFrame : 매 프레임마다 큐브를 회전
+    useFrame(() => {
+        if (cubeRef.current) {
+            cubeRef.current.rotation.y += rotationSpeed; // Y축으로 회전
+            cubeRef.current.rotation.x += rotationSpeed / 2; // X축으로 회전
+        }
+    });
+
+    const onMouseDown = (e: React.MouseEvent) => {
```

ESLint

[error] 'e' is defined but never used.

(no-unused-vars)